zerver/lib/actions.py

```diff
@@ -4366,10 +4366,39 @@ def do_send_create_user_group_event(user_group: UserGroup, members: List[UserPro
                  )
     send_event(event, active_user_ids(user_group.realm_id))
 
-def check_add_user_group(realm: Realm, name: Text, initial_members: List[UserProfile],
+def send_user_group_update_notification(acting_user: UserProfile, group_name: Text,
+                                        members: List[UserProfile], action: Text) -> None:
+    if action == "add_members":
+        message_template = """Hi there! We thought you'd like to know that {} ({}) \
+added you to the user group **{}**."""
+    elif action == "remove_members":
+        message_template = """Hi there! We thought you'd like to know that {} ({}) \
```

Comment:
You have been removed from @*{}*. You can contact an organization administrator to be re-added. 

(Is that our actual policy? Or can people re-add themselves? You can always set an alert word for `@*group name*` anyway, right?)